### PR TITLE
Added miqAjaxTimers equiv back in

### PR DESCRIPTION
**Pull request by psav**

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit af0f19ac4279257164a413bcd7ba7bc09ded3ad4:

Everything seems all right :smile: .
*CFME QE Bot*

File: data/patches/miq_application.js.diff [hunk 1]
```diff
@@ -21,7 +21,7 @@
 +function checkMiqQE(key) {
 +  // Returns 0 if key is undefined or set to 'complete', otherwise 1
 +  var v = getMiqQE(key);
-+  return (v === undefined || v === 'complete') ? 0 : 1;
++  return (v === undefined || v === 'complete' || v === 0 ) ? 0 : 1;
```
Comment (mfalesni) [inline data/patches/miq_application.js.diff]: I would personally do:
```javascript
return (v === undefined || v === 'complete' || v === 0 ) ? 0 : v;
```

so we can see the number in the automation.

**Comment by dajoRH** [comment]
Lint report for commit 2d8368bf46b1ec61fc0a423ade8583145f4c307b:

Everything seems all right :smile: .
*CFME QE Bot*

File: data/patches/miq_application.js.diff (hunk 1 above)
Comment (psav) [inline data/patches/miq_application.js.diff]: As discuseed this can't be done

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

**Pull request closed without merging**